python/ray/dataframe/dataframe.py

```diff
@@ -78,6 +78,10 @@ def __init__(self, data=None, index=None, columns=None, dtype=None,
             col_metadata (_IndexMetadata):
                 Metadata for the new dataframe's columns
         """
+        if isinstance(data, DataFrame):
+            self._data = data._data
```

Comment:
Because of the collision with `pandas.DataFrame._data`, I would prefer this name to be changed. Our `_data` would not have the same behavior as `pandas` and if there are tools/code that rely on accessing `_data` in a `DataFrame`, we should not return this object.

Reply:
resolved, thanks!